app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -191,6 +193,10 @@ snippet: true
     done
   <% end %>
 
+  <% if host_param_true?('subscription_manager_auto_attach', false) -%>
```

Comment:
This is safe to do in general. When the host's organization is in Simple Content Access mode it's an unnecessary step, but not harmful:

```
[root@centos7-katello-client-a vagrant]# subscription-manager attach --auto
Ignoring request to auto-attach. It is disabled for org "Default_Organization" because of the content access mode setting.
[root@centos7-katello-client-a vagrant]# echo $?
0
```